src/connection/mc_connection_man.erl

```diff
@@ -76,17 +132,26 @@ process_error(_, Doc) ->
   erlang:error({bad_query, Doc}).
 
 %% @private
--spec request_raw(port(), mc_worker_api:database(), mongo_protocol:message(), module()) ->
+-spec request_raw(any(), mc_worker_api:database(), mongo_protocol:message(), module()) ->
```

Comment:
Just curiosity: what can arrive here as `Socket` other than `port()`?

Reply:
`ssl:sslsocket()` is `any()`.